cache/dummycache/dummycache.go

```diff
@@ -38,11 +38,6 @@ func (s *accountService) Get(id string) (*cache.Account, error) {
 	}, nil
 }
 
-// Set will always return nil since this is a dummy service
-func (s *accountService) Set(account *cache.Account) error {
-	return nil
-}
```

Comment:
I was considering renaming this concept, when I noticed all implements just return `nil` and is never called.